src/main/java/com/faforever/client/map/generator/GithubGeneratorRelease.java

```diff
@@ -0,0 +1,13 @@
+package com.faforever.client.map.generator;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+public class GithubGeneratorRelease {
+  private String tag_name;
```

Comment:
Don't break the Java naming convention

```suggestion
  @JsonProperty("tag_name")
  private String tagName;
```

Reply:
Done